build_projects/dotnet-host-build/PackageTargets.cs

```diff
@@ -49,6 +50,41 @@ public static BuildTargetResult GenerateVersionBadge(BuildTargetContext c)
         }
         
         [Target]
+        [BuildPlatforms(BuildPlatform.Windows)]
+        public static BuildTargetResult CopyNativeSymbols(BuildTargetContext c)
+        {
+            var nativeSymbolsRoot = Path.Combine(Dirs.Output, "obj", "nativeSymbols");
+            var nativeSymbolsZipFileDir = Path.Combine(Dirs.Intermediate, "nativeSymbols");
+
+            Utils.DeleteDirectory(nativeSymbolsRoot);
+            Directory.CreateDirectory(nativeSymbolsRoot);
+
+            Utils.DeleteDirectory(nativeSymbolsZipFileDir);
+            Directory.CreateDirectory(nativeSymbolsZipFileDir);
+
+            foreach (var file in Directory.GetFiles(Dirs.CorehostLatest, "*.pdb", SearchOption.TopDirectoryOnly))
```

Comment:
https://github.com/dotnet/core-setup/blob/master/build_projects/shared-build-targets-utils/Utils/SharedFrameworkPublisher.cs#L171 

https://github.com/dotnet/core-setup/blob/master/build_projects/dotnet-host-build/PackageTargets.cs#L101 

Locked bits are published in the zips.